ts/lib/transporters.ts

```diff
@@ -19,7 +19,18 @@ import * as request from 'request';
 // tslint:disable-next-line
 const pkg = require('../package.json');
 
-export interface Transporter { request(opts, opt_callback): any; }
+export interface Transporter {
+  request(opts: any, opt_callback: RequestCallback): any;
+}
+
+export interface RequestCallback {
```

Comment:
Multiple people have lost multiple hours of debugging time because of this callback. Can we call this something that people don't easily mistake for `request.RequestCallback`?

Perhaps `BodyResponseCallback` or something like that?